### ISSUE TITLE
Installation schlägt fehl

### ISSUE THREAD

**Issue by Punktraum** [codefog/contao-member_export#9]
Ich hatte die Erweiterung auf einer älteren Installation seit langem im Einsatz. Diese Installation habe ich nun auf Contao 4 (Version 4.11.9, Contao Manager 1.4.10, PHP 8) migriert.
Beim Versuch, die Erweiterung zu installieren, erhalte  ich folgende Meldung im Composer:

`composer update codefog/contao-member_export contao/conflicts`
```
> Resolving dependencies using Composer Cloud v3.2.0
!!! Current server is sponsored by: Contao Association !!!
[7.2MiB/0.18s] Loading composer repositories with package information
[62.3MiB/2.12s] Updating dependencies
[1499.6MiB/13.31s] Your requirements could not be resolved to an installable set of packages.
[1499.6MiB/13.32s] 
  Problem 1
    - codefog/contao-member_export[1.0.1, ..., 1.0.4] require contao-community-alliance/composer-plugin 2.* -> satisfiable by contao-community-alliance/composer-plugin[2.0.0, ..., 2.4.3].
    - contao-community-alliance/composer-plugin[2.0.0, ..., 2.3.3] require composer-plugin-api 1.0.0 -> found composer-plugin-api[2.1.0] but it does not match the constraint.
    - contao-community-alliance/composer-plugin[2.3.4, ..., 2.4.3] require composer-plugin-api ^1.0 -> found composer-plugin-api[2.1.0] but it does not match the constraint.
    - Root composer.json requires codefog/contao-member_export ^1.0 -> satisfiable by codefog/contao-member_export[1.0.1, 1.0.2, 1.0.3, 1.0.4].
[1499.6MiB/13.32s] <warning>Running update with --no-dev does not mean require-dev is ignored, it just means the packages will not be installed. If dev requirements are blocking the update you have to resolve those problems.</warning>
[77.7MiB/14.88s] Memory usage: 77.67MB (peak: 1500.68MB), time: 14.89s.
[77.7MiB/14.88s] Finished Composer Cloud resolving.
# Cloud Job ID zmNDSWhmx3bUZx1T2YEdthfn07z97G1DVVihj68XrOJWySLbWxhM4tMt97Jo7w6o5TR16wYtCgGQ failed
```

**Comment by Punktraum** [comment]
Would it be better in English?

I had been using the extension on an older installation for a long time. I have now migrated this installation to Contao 4 (version 4.12.6, Contao Manager 1.4.10, PHP 8).
When trying to install the extension, I get the following message in the composer:

`> Resolving dependencies using Composer Cloud v3.2.0
!!! Current server is sponsored by: Contao Association !!!
[7.2MiB/0.18s] Loading composer repositories with package information
[62.3MiB/2.12s] Updating dependencies
[1499.6MiB/13.31s] Your requirements could not be resolved to an installable set of packages.
[1499.6MiB/13.32s] 
  Problem 1
    - codefog/contao-member_export[1.0.1, ..., 1.0.4] require contao-community-alliance/composer-plugin 2.* -> satisfiable by contao-community-alliance/composer-plugin[2.0.0, ..., 2.4.3].
    - contao-community-alliance/composer-plugin[2.0.0, ..., 2.3.3] require composer-plugin-api 1.0.0 -> found composer-plugin-api[2.1.0] but it does not match the constraint.
    - contao-community-alliance/composer-plugin[2.3.4, ..., 2.4.3] require composer-plugin-api ^1.0 -> found composer-plugin-api[2.1.0] but it does not match the constraint.
    - Root composer.json requires codefog/contao-member_export ^1.0 -> satisfiable by codefog/contao-member_export[1.0.1, 1.0.2, 1.0.3, 1.0.4].
[1499.6MiB/13.32s] <warning>Running update with --no-dev does not mean require-dev is ignored, it just means the packages will not be installed. If dev requirements are blocking the update you have to resolve those problems.</warning>
[77.7MiB/14.88s] Memory usage: 77.67MB (peak: 1500.68MB), time: 14.89s.
[77.7MiB/14.88s] Finished Composer Cloud resolving.
# Cloud Job ID zmNDSWhmx3bUZx1T2YEdthfn07z97G1DVVihj68XrOJWySLbWxhM4tMt97Jo7w6o5TR16wYtCgGQ failed`

**Comment by qzminski** [comment]
The extension is not yet compatible with that version of Contao and PHP 8. I have plans to update it next week.

**Comment by Punktraum** [comment]
Thanks - sounds good!

**Comment by qzminski** [comment]
I have added the support for PHP 8 and Contao 4.9+, can you try updating the extension in Contao Manager using `dev-develop@dev` version for that?

**Comment by Punktraum** [comment]
Thank you very much!
The installation works and so does the export of the members. Great!

**Comment by qzminski** [comment]
Great, thank you for testing! I have just released 2.3.0 stable.
